styles/style.css

```diff
@@ -230,6 +231,7 @@ div#content_block { width: 70% }
 .rtsocial-twitter-vertical-button { text-align: center; }
 .rtsocial-twitter-button,
 #rtsocial-ord-tw label { background-position: 0 -557px;  display: block; float: left; height: 21px; margin: 0; padding: 0; width: 58px; text-indent: -9999px; }
+.rtsocial-twitter-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -557px no-repeat !important;}
```

Comment:
Use of `!important` is causing the PHPCS test failure.

Reply:
The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files